openedx/stanford/cms/envs/common.py

```diff
@@ -46,6 +46,10 @@
     # Log out sneakpeek users
     'sneakpeek.middleware.SneakPeekLogoutMiddleware',
 )
+OPTIONAL_APPS = (
```

Comment:
Now we're redefining the entire tuple;
instead, we want to append to it,
else we'll nuke everything edx has added.